Tests/Unit/MSFT_xWebsite.Tests.ps1

```diff
@@ -1236,19 +1236,19 @@ try
 
                 $MockBindingInfo = @(
                     New-CimInstance -ClassName MSFT_xWebBindingInformation -Namespace root/microsoft/Windows/DesiredStateConfiguration -Property @{
-                        Protocol              = 'http'
+                        Protocol              = 'https'
                         IPAddress             = '*'
-                        Port                  = 8080
+                        Port                  = 443
```

Comment:
I'm unclear why this needed to change, why are we changing it from 8080 to 443, and adding in a thumbprint etc? Does the test not catch this as is?